florestad/src/main.rs

```diff
@@ -144,7 +188,20 @@ fn main() {
                 json_rpc_address: rpc_address,
                 electrum_address,
             };
-
+            let data_dir = &ctx
+                .data_dir
+                .clone()
+                .or_else(|| Some(".".to_string()))
```

Comment:
The default here is inconsistent, I think you can move this to inside `run_with_ctx` where you have a sane `data_dir` available.

Reply:
done